### PR TITLE
🧻 Julia Logging in browser console

### PR DESCRIPTION
**Pull request by fonsp**
A basic version of #437. It relays all Logging messages to the browser console (and also to the original terminal).

That PR will take a lot of work to get right, and it would be nice to have something simple before it it finished.

### CODE REVIEW

**Comment by fonsp** [comment]
Log messages are visible in the browser console:

image

And as a quick hack they contain the line node so that you can hover to see where it came from:

image

**Comment by fonsp** [comment]
@c42f @shashi @dralletje

**Review by c42f — COMMENTED** [review]
Very cool! I suggest the :scroll: emoji for logging :laughing: 

Mapping to the javascript console logger is a neat way to get UI functionality without a lot of up front work.

I suspect debug logging may be too verbose to use by default (as other packages may emit debug messages). I suggest we restrict debug messages to come from the user's notebook only.

File: src/evaluation/WorkspaceManager.jl [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-import ..Pluto: Configuration, Notebook, Cell, ServerSession, ExpressionExplorer, pluto_filename, trycatch_expr, Token, withtoken, tamepath, project_relative_path
+import ..Pluto: Configuration, Notebook, Cell, ServerSession, ExpressionExplorer, pluto_filename, trycatch_expr, Token, withtoken, tamepath, project_relative_path, putnotebookupdates!, UpdateMessage
 import ..Configuration: CompilerOptions
 import ..PlutoRunner
 import Distributed
 
 "Contains the Julia process (in the sense of `Distributed.addprocs`) to evaluate code in. Each notebook gets at most one `Workspace` at any time, but it can also have no `Workspace` (it cannot `eval` code in this case)."
 mutable struct Workspace
     pid::Integer
+    log_channel::Distributed.RemoteChannel
```
[the remark this answers is not in the document]
Comment (c42f) [reply src/evaluation/WorkspaceManager.jl:11]: Oh wait, this is the `recv` side? Seems good in that case.

**Pull request merged**